api/api.py

```diff
@@ -344,11 +344,11 @@ def load(path: str, *args):
     hi() # prints "Hello world!"
   """
 
-def local(command: str, quiet: bool = False, command_bat: str = "") -> Blob:
+def local(command: Union[str, List[str]], quiet: bool = False, command_bat: str = "") -> Blob:
```

Comment:
@nicks afaict the point of `command_bat` is to make sure the command gets executed as `["cmd", "/S", "/C", cmd]`, right? Seems to me that if you've got a windows Tilt install, if you pass a string to `command` it goes through the same codepath as `command_bat` -- are these two actually redundant or am i missing something?

Reply:
here's an example that might help:
https://github.com/tilt-dev/tilt/blob/7dadfff0fa68c929932a20e7ecf72444b0595378/internal/tiltfile/tiltfile_test.go#L287

This kind of API isn't totally unexpected if you're used to other cross-platform tools that need to run scripts on windows:
https://docs.bazel.build/versions/master/be/general.html#genrule.cmd_bat

Reply:
aaaah I see, it's not because you want the same command you passed to `cmd`, but handled differently; it's because you want to, within a single tiltfile, provide one command to run if it's osx/linux and one command if it's windows. 👍 

Still, it's the case that `command_bat` values go through the same `valueToCmdHelper` as our other commands, and so can be passed as either string or array. do you want to / should I update the docs to reflect that?